grails-app/i18n/messages_es.properties

```diff
@@ -2202,6 +2202,10 @@ profile.emailNotifications.title = Alertas por email
 profile.leftMenu.title = Configuración
 profile.menu.changeMail = Cambiar correo electronico
 profile.menu.changePass = Contraseña
+profile.menu.adminPrivateFiles = Ficheros privados del admin
+profile.menu.adminPrivateFiles.title = Ficheros
+profile.menu.adminPrivateFiles.subtitle = Gestión de ficheros
```

Comment:
Those texts are in POE translated to english and spanish?

Reply:
Yes, they are in english, spanish and catalan